### ISSUE TITLE
How to only update the final FC layer before the softmax layer in a pre-trained CNN model?

### ISSUE THREAD

**Issue by tzczsq** [vlfeat/matconvnet#557]
Hi,  I have a pre-trained CNN model, like AlexNet model. I want to only update the final FC layer (fc7) before the softmax layer in this model.  Note that I have produced some new feature data with 4096 dimension corresponding to the final FC layer (fc7). So, I will employ these produced 4096-D feature data to update the final FC layer (fc7).
Could you please tell me how to realize this?

**Comment by pxnguyen** [comment]
You can use the backpropDepth variable.

**Comment by fengyunxiaozi** [comment]
set the learning rates of the layers that you do not want to update to 0
